### PR TITLE
BRN-14: Configure checkstyle

### PR DESCRIPTION
**Pull request by valeriipestov**

### CODE REVIEW

**Review by azarot — APPROVED** [review]

File: config/checkstyle/suppressions.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+<suppressions>
+    <!--    <suppress checks="JavadocStyleCheck"-->
+    <!--              files="AbstractComplexityCheck.java"-->
+    <!--              lines="82,108-122"/>-->
+    <!--    <suppress checks="MagicNumberCheck"-->
+    <!--              files="JavadocStyleCheck.java"-->
+    <!--              lines="221"/>-->
+    <!--    <suppress message="Missing a Javadoc comment"/>-->
+</suppressions>
```
Comment (maingroon) [inline config/checkstyle/suppressions.xml:21]: Commented block need here?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: 'CI'
-on: [push, pull_request]
+on: [ push, pull_request ]
```
[the remark this answers is not in the document]
Comment (valeriipestov) [reply .github/workflows/ci.yml:2]: No, It was IDEA's autoformatting

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -20,7 +20,9 @@ jobs:
       - name: Make gradlew executable
         run: chmod +x ./gradlew            # make ./gradlew an executable command for Linux
 
+      - name: Execute checkstyle
+        run: ./gradlew checkstyleMain     # run gradle checkstyle
```
[the remark this answers is not in the document]
Comment (valeriipestov) [reply .github/workflows/ci.yml:24]: I've added restrictions to run the checkstyle automatically so added this point.
restrictions are in build.gradle file:

https://github.com/maingroon/svydovets-bring/pull/3/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R49

https://github.com/maingroon/svydovets-bring/pull/3/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R54

**Review by EEM86 — APPROVED** [review]

File: config/checkstyle/suppressions.xml (hunk 1 above)
Comment (EEM86) [reply config/checkstyle/suppressions.xml:21]: I think it is ok to leave it. It is an  example how to supress checkstyle error.

**Pull request merged**